tests/tests.py

```diff
@@ -230,6 +229,7 @@ def test_single_form_csv_with_rep_fields_start_with_repeat(self):
         for line1, line2 in izip(open(get_filename('models.py'),'r'),open(get_filename(cmp_fileName),'r')):
             self.assertEqual(line1,line2)
     def test_single_form_csv_with_rep_fields_many_nested(self):
```

Comment:
I am not sure what you mean that setup.py calls the method. The underlying test runner is what determines and executes the methods on a `TestCase` class. Commenting it out will just make it invisible and therefore not be called when the test suite runs.